app/controllers/users_controller.rb

```diff
@@ -26,6 +26,10 @@ def update
     end
   end
 
+  def show
+    @loans = Loan.where(member_id: @user.id)
```

Comment:
I would suggest using `@user.loans`. (you may need to add the `has_many` in `User` because I do not think it is there).

Reply:
I updated this.

Reply:
It looks like you did add the `user_id` column to the loans. Now it has both `member_id` and `user_id`, which is even more confusing. Can we clean this up?